generated/niscope/session.py

```diff
@@ -1899,6 +1914,97 @@ def configure_vertical(self, range, coupling, offset=0.0, probe_attenuation=1.0,
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    def fetch(self, timeout=datetime.timedelta(seconds=5.0), num_samples=None, relative_to=enums.FetchRelativeTo.PRETRIGGER, offset=0, record_number=0, num_records=-1):
+        '''fetch
+
+        Returns the waveform from a previously initiated acquisition that the
+                        digitizer acquires for the specified channel. This method returns
```

Comment:
indentation is odd here. How does this affect things?

Reply:
It doesn't. It is cosmetic, but now fixed.